index.html

```diff
@@ -63,5 +63,43 @@ <h1>
     
         </div>
     </section>
+
+   <!-- ------------------------------------mobile section-------------------- -->
+<section class="section-nav">
+
+   <header class="header-mobile">
+    <nav>
+        <img src="images/section_header.png" alt="logo">
+    </nav>
+   </header>
+
+<section class="section-mobile">
+   <div class="project-title">
+    <h1>Tonic</h1>
```

Comment:
- [ ] Due to SEO concerns, only one `h1` tag is allowed per document, please switch this `h1` to a different `hx` tag. Tip: think about HTML as a book, h1 is like the book title, h2 could be the chapters title, and so on.